lexibank_constenlachibchan.py

```diff
@@ -26,33 +31,39 @@ class Dataset(BaseDataset):
     id = "constenlachibchan"
     language_class = CustomLanguage
     lexeme_class = CustomLexeme
+    concept_class = CustomConcept
 
     def cmd_makecldf(self, args):
         args.writer.add_sources()
-        concepts = {}
-        for concept in self.concepts:
-            idx = concept['NUMBER']+'_'+slug(concept['ENGLISH'])
+
+        concept_lookup = {}
+        for concept in self.conceptlists[0].concepts.values():
+            idx = concept.id.split("-")[-1] + "_" + slug(concept.attributes["spanish"])
             args.writer.add_concept(
                     ID=idx,
-                    Name=concept['ENGLISH'],
-                    Concepticon_ID=concept['CONCEPTICON_ID'],
-                    Concepticon_Gloss=concept['CONCEPTICON_GLOSS']
+                    Name=concept.attributes["spanish"],
```

Comment:
concept.gloss is better here.

Reply:
Would you suggest adding `SPANISH` as an additional column instead?